src/share/share.validator..ts

```diff
@@ -1,13 +0,0 @@
-import Validator from '../validator';
```

Comment:
Y U do dis?

Reply:
Yonatan told me to O_O

Reply:
![OK](https://media.giphy.com/media/5hc2bkC60heU/giphy.gif)